pkg/console/operator/sync_v400.go

```diff
@@ -593,3 +602,14 @@ func (co *consoleOperator) GetAvailablePlugins(enabledPluginsNames []string) []*
 	}
 	return availablePlugins
 }
+
+func getNodeArchitectures(nodes *corev1.NodeList) []string {
+	nodeArchitecturesSet := sets.NewString()
+	for _, node := range nodes.Items {
+		nodeArch := node.Labels[api.NodeArchitectureLabel]
+		nodeArchitecturesSet.Insert(nodeArch)
+	}
+	nodeArchitectures := nodeArchitecturesSet.List()
+	sort.Strings(nodeArchitectures)
```

Comment:
@jhadvig Per @joelanford this can be removed

https://github.com/openshift/console-operator/pull/669#discussion_r951778015

Sorry for the noise

Reply:
@spadgett np. Updated :)